test/test.mk

```diff
@@ -31,9 +33,13 @@ test:
 test-sh:
 	$(test-dir)/env $(base) $(mode) $(binaries) -i /bin/sh
 
+test-spec:
+	$(spec-dir)/generate_spec.py -d $(test-dir)/case
+	asciidoctor-pdf --theme $(spec-dir)/theme.yml -a pdf-fontsdir=$(spec-dir)/fonts -o $(test-specification) $(spec-dir)/Readme.adoc
```

Comment:
Please add the [LICENSE.md](https://github.com/adobe-fonts/source-code-pro/blob/release/LICENSE.md) file to the fonts directory, they are free to use, but we still need to do our due diligence.

Please also mention the upstream reference/URL in the commit message that imports the fonts.  E.g., "Fonts imported from https://github.com/adobe-fonts/source-code-pro/"